### ISSUE TITLE
Style/BitwisePredicate iinspection error

### ISSUE THREAD

**Issue by JoeCohen** [rubocop/rubocop#13459]
An error occurred while Style/BitwisePredicate cop was inspecting a file when running the command below. 
The file is `app/classes/api2/parsers/boolean_parser.rb:8:14`. 

```ruby
% rubocop --auto-gen-config --no-exclude-limit
```

--------

## Expected behavior

Not throw an error.

## Actual behavior

An error occurred while Style/BitwisePredicate cop was inspecting mushroom-observer/app/classes/api2/parsers/boolean_parser.rb:8:14.

Mention the following information in the issue report:
1.68.0 (using Parser 3.3.6.0, rubocop-ast 1.36.1, analyzing as Ruby 3.3, running on ruby 3.3.0) [x86_64-darwin23]

## Steps to reproduce the problem

- Clone https://github.com/MushroomObserver/mushroom-observer
- `rubocop --auto-gen-config --no-exclude-limit`

## RuboCop version

```
% rubocop -V
1.68.0 (using Parser 3.3.6.0, rubocop-ast 1.36.1, analyzing as Ruby 3.3, running on ruby 3.3.0) [x86_64-darwin23]
  - rubocop-performance 1.23.0
  - rubocop-rails 2.27.0
```

**Comment by vlad-pisanov** [comment]
This was fixed in https://github.com/rubocop/rubocop/pull/13411
Fix is on master branch
